src/config.ts

```diff
@@ -373,6 +374,8 @@ export function stripAndRedirectConfig(config: Config) {
   // affect header labels, which also uses `title` directive to implement.
   redirectConfig(config, 'title', 'group-title');
 
+  // Remove background config since we apply it to the spec.
```

Comment:
Why is this comment here, not above in `VL_ONLY_CONFIG_PROPERTIES`?

Reply:
Good catch. Fixed.